webapp/app/admin/replications/new/template.hbs

```diff
@@ -16,6 +16,9 @@
       <label>URL or hostname</label>
       {{input class="form-control" value=model.url placeholder="URL or hostname" required=true}}
     </div>
+    <div class="form-group">
+      {{input type="text" class="form-control" value=model.index_name placeholder="Index name" required=true}}
```

Comment:
Provide a default value here. I think `backslash` is a reasonable default, and could also stay the default at the DB level